app/views/games/show.html.erb

```diff
@@ -1,29 +1,48 @@
-<div class= "title"><h2><%= @game.name %></h2></div>
-<h1><%= @game.name %></h1>
-<div><%= image_tag(@game.photo_url, size:"300") %></div>
-<p>Players: <%= @game.min_players %> to <%= @game.max_players %></p>
-<p>Playing time: <%= @game.playing_time %></p>
-<p>Complexity: <%= @game.complexity_name %></p>
-<p><%= @game.description %></p>
-<%= link_to "edit game", edit_game_path(@game) %>
+<div class= "title">
+  <h1><%= @game.name %></h1>
+</div>
+<div class="row">
+<div class=""></div>
+<ul class="large-block-grid-2">
+  <li>
+    <ul>
+      <li class="larger">Players: <%= @game.min_players %> to <%= @game.max_players %></li>
+      <br>
+      <li class="larger">Playing time: <%= @game.playing_time %></li>
+      <br>
+      <li class="larger">Complexity: <%= @game.complexity_name %></li>
+      <br>
+      <li>
+        <h4>Description:</h4>
+        <%= @game.description %>
+      </li>
+    </ul>
+  </li>
+  <li><%= image_tag(@game.photo_url, size:"400") %></li>
+  <%= link_to "edit game", edit_game_path(@game) %>
+</ul>
 <h2>what the gamers say</h2>
```

Comment:
Review your capitalization -- be consistent with either all lower case or normal case or whatever. Pick one standard and stick to it throughout your views.